app/services/profile_fields/import_from_csv.rb

```diff
@@ -0,0 +1,21 @@
+module ProfileFields
+  class ImportFromCsv
+    HEADERS = %i[label input_type placeholder_text description].freeze
```

Comment:
The CSV files we import don't need a header row. I also made myself a note to properly document all of this, but first I want to get the implementation nailed down.